library/src/main/java/ch/decent/sdk/model/AssetWithAmount.kt

```diff
@@ -0,0 +1,18 @@
+package ch.decent.sdk.model
+
+import java.text.NumberFormat
+
+data class AssetWithAmount(
```

Comment:
unimportant nitpick, but I'd probably call it `AmountWithAsset`. I think the more important part is the amount and asset is there only to enable formatting etc.

Reply:
makes sense